csharp/src/Ice/PropertiesAdmin.cs

```diff
@@ -7,9 +7,9 @@
 
 namespace Ice
 {
-    /// <summary>Implementations of the Properties admin facet implement both IPropertiesAdmin (which corresponds
-    /// to the Slice interface PropertiesAdmin) and the local INativePropertiesAdmin interface.</summary>
-    public interface INativePropertiesAdmin
+    /// <summary>This partial interface extends the IPropertieAdmin interface generated from the Slice interface
+    /// PropertiesAdmin. All Properties admin facet implement this interface.</summary>
```

Comment:
`All Properties admin facet implement this interface`
I don't understand this sentence, what do you mean?

Reply:
I removed it as it's indeed unclear here.